libs/navigation/bootstrapper.js

```diff
@@ -12,6 +10,19 @@ export default async function bootstrapBlock(miloConfigs, blockConfig) {
     const block = createTag(targetEl, { class: name });
     document.body[blockConfig.appendType](block);
   }
+  // Configure Unav components and redirect uri
+  if (blockConfig.targetEl === 'header') {
+    ['unavComponents', 'redirect'].forEach((key) => {
+      if (blockConfig[key]) {
+        const metaTag = createTag('meta', {
+          name: key === 'unavComponents' ? 'universal-nav' : 'adobe-home-redirect',
+          content: blockConfig[key],
+        });
+        document.head.append(metaTag);
+      }
+    });
```

Comment:
**Just a suggestion**: we can use an array `metaTagConfigs` to store the mapping of keys to meta tag names. This makes it easier to extend or modify the configuration as needed and avoids using the ternary condition within the `forEach` loop.
```suggestion
    const metaTagConfigs = [
      { key: 'unavComponents', name: 'universal-nav' },
      { key: 'redirect', name: 'adobe-home-redirect' }
    ];
    metaTagConfigs.forEach(({ key, name }) => {
      if (blockConfig[key]) {
        const metaTag = createTag('meta', {name, content: blockConfig[key]});
        document.head.append(metaTag);
      }
    });
```

Reply:
Yes, this was also my first approach, but I wanted to avoid creating an array of objects. However, since the theme is also coming up, it makes sense to update it this way.